### PR TITLE
Enabling Disk Encryption

### PR DESCRIPTION
**Pull request by kylerschin**

### CODE REVIEW

**Comment by kylerschin** [comment]
@cassidyjames can plz have merge?

**Comment by cassidyjames** [comment]
Assigning over to @ryanleesipes since it's content-related. Thanks for the contribution!

**Comment by Xeboc** [comment]
@kylerschin We will probably want to have the images locally in the repo rather than at githubusercontent.com.  @cassidyjames can confirm, but I think the images should be included in the pull request in a local repo folder.  Less volatility that way.

**Comment by kylerschin** [comment]
@Xeboc Okaly Dokaly, Neighborino!

**Comment by kylerschin** [comment]
@Xeboc @cassidyjames Every is Okaly Dokaly Fine!

**Comment by kylerschin** [comment]
@cassidyjames @ryanleesipes  ya wanna merge me?

**Comment by kylerschin** [comment]
@Xeboc Hello?

**Comment by kylerschin** [comment]
@ryanleesipes anyone?

**Comment by kylerschin** [comment]
@cassidyjames @ryanleesipes Hello? This has been sitting here for 4 days!

File: docs/disk-encryption-luks.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+![image](/images/disk-encryption-luks/enable-encryption.png)
+
+Next, choose an encryption password. Keep this safe!
+
+![image](/images/disk-encryption-luks/enter-password.png)
+
+You may want to over-write your empty space if this is a re-install, so if anybody ever does get a hold of your drive, it won't be possible to take your data.
+
+Your done!
```
Comment (cassidyjames) [inline docs/disk-encryption-luks.md]: You're

File: images/LICENSE [hunk 2]
```diff
@@ -0,0 +1,97 @@
+Creative Commons Attribution 4.0 International Public License
```
Comment (cassidyjames) [inline images/LICENSE]: We have a LICENSE file at the top-level of the project; this file should be omitted.

**Comment by ryanleesipes** [comment]
@kylerschin once the changes @cassidyjames has requested are implemented, this can be merged.

File: docs/disk-encryption-luks.md (hunk 1 above)
Comment (kylerschin) [inline docs/disk-encryption-luks.md]: Ooppps.... Typing WAAAYYYYYYYYY to fast.

**Comment by kylerschin** [comment]
Changes have been made.

**Pull request merged**